ores/scoring_systems/scoring_system.py

```diff
@@ -38,14 +38,13 @@ def check_context_models(self, request):
         if len(missing_models) > 0:
             raise MissingModels(request.context_name, missing_models)
 
-    def score(self, request, ip=None):
+    def score(self, request):
         self.check_context_models(request)
 
         locked = None
-        if ip and (not request.precache) and self.lock_manager is not None:
-            connected = self._connect_to_lock_manager()
-            if connected:
-                locked = self._lock_ip(ip.strip())
+        if request.ip and (not request.precache) and \
```

Comment:
Yeah, we discussed this halfak and decided to use precache flag instead of the whitelisted IPs because all of DoSes we have had until now were not intentional (they weren't malicious)